src/utils/parser.ts

```diff
@@ -0,0 +1,146 @@
+type CsvData = {
+  name: string;
+  title?: string;
+  description?: string;
+  type: string;
+  minimum?: string;
+  maximum?: string;
+  unit?: string;
+  href: string;
+  "modbus:unitID": number;
+  "modbus:address": string;
+  "modbus:quantity": string;
+  "modbus:type"?: string;
+  "modbus:zeroBasedAddressing": string;
+  "modbus:entity": string;
+  "modbus:pollingTime"?: string;
+  "modbus:function": string;
+  "modbus:mostSignificantByte": string;
+  "modbus:mostSignificantWord": string;
+  "modbus:timeout"?: string;
+};
+
+type PropertyForm = {
+  op: string | string[];
+  href: string;
+  "modbus:unitID": number;
+  "modbus:address": number;
+  "modbus:quantity": number;
+  "modbus:type"?: string;
+  "modbus:zeroBasedAddressing": boolean;
+  "modbus:entity": string;
+  "modbus:pollingTime"?: string;
+  "modbus:function": string;
```

Comment:
can be made optional for now. It basically has default behavior but it should not be filled with `""`